multi-node/aws/pkg/cluster/cluster_test.go

```diff
@@ -300,6 +321,7 @@ hostedZoneId: /hostedzone/staging_id_3  #non-existant hostedZoneId
 		clusterConfig, err := config.ClusterFromBytes([]byte(configBody))
 		if err != nil {
 			t.Errorf("could not get valid cluster config: %v", err)
+			continue
```

Comment:
Yep. This error indicates an error parsing a single config, doesn't necessarily mean any others will fail.